style_guide.md

```diff
@@ -0,0 +1,118 @@
+# Style Guide
+
+## What is this?
+This is a set of guidelines on how to *style* code you plan on contributing to the prog-o-meter repository. You don't have to follow each, or any of them, but it may take your PR longer to get merged if someone else has to rewrite and document your code.
+
+## Why is this useful? 
+When everybody is on the same page, everyone writes better code. The purpose of the prog-o-meter project is too get newcomers out of their shell and excited to start learning, and feel confident contributing to the open source community. The faster everyone starts getting comfortable the better the project will be.
+
+## 1. Quick Start
+These are some quick guidelines and principles that this project follows. 
+
+In general...
+* **Structure** should follow the [PEP 8](https://www.python.org/dev/peps/pep-0008/#introduction) format.
+* **Docstrings** should follow the [Google Python](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) format. (Sec. 3)
+* Code and comments should be kept **beginner friendly**. Purpose and diction should be easy to understand. (Sec. 4)
```

Comment:
What do you mean with the word diction?

Reply:
Diction here is talking about the choice of words in comments specifically. Comments that use terminology new programmers aren't familiar with with will only further confuse them. I could add an example here.

Reply:
I think the message is a good one, but do you think there is another word than diction that we could use? I am not sure that people who don't have English as their first language (like me) will understand the intention. I think adding an example also, like you suggest, would be great.

Reply:
Changed "diction" to "word choice". Since there are multiple examples of proper comments in section 3 the guide now explicitly says to refer to it for examples. I think this is the most elegant solution rather than adding examples into the "Quick Start" section.